src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -36,17 +39,17 @@
 
     static class SensitivityVariableConfiguration {
```

Comment:
Maybe we should remove SensitivityVariableConfiguration? I added it to avoid computing same sensibility multiple times but there is no reason for that to happen frequently (user input mistake?) and it adds useless complexity.
So instead of `Map<SensitivityVariableConfiguration, SensitivityFactorGroup> factorsByVarConfig` we would only have `List<SensitivityFactor<?, ?>>`

Reply:
I feel like SensitivityVariableConfigurations are useful when the user gives a provider that contains all the generators of the network. Not sure if this is considered as an "user input mistake". But in this case, the SensitivityVariableConfiguration is useful to remove duplicate (generators connected on the same bus).

Reply:
Ok so let s keep it